contrib/destination-starrocks/src/main/java/io/airbyte/integrations/destination/starrocks/StarRocksBufferedConsumerFactory.java

```diff
@@ -1,16 +1,20 @@
-// Copyright 2021-present StarRocks, Inc. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This project is not a project of ASF, so we don't use this license header.

```
/*
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

The above one should be the right one.

By the way, why do you change the original license header?

Reply:
My bad, this code was originally forked from https://github.com/liuzhongjun89/starrocks-airbyte-connector which had the license change

I will update the license

Reply:
I've opened an issue about it on liu's repository.
https://github.com/liuzhongjun89/starrocks-airbyte-connector/issues/15